mod/deckorate_core/set/type/metric/export.rb

```diff
@@ -108,3 +108,65 @@ def semicolon_separated_values column
     card.send("#{column}_card").item_names.join ";"
   end
 end
+
+format :jsonld do
+  
+  def molecule
+    metric_jsonld(atom)
+  end
+
+  private
+
+    def metric_jsonld(a)
+    {
+      "@context" => "#{request.base_url}/context/#{card.type}.jsonld",
+      "@id"      => path(mark: card.name, format: nil),
+      "@type"    => card.type,
+
+      "title"    => a[:title] || a["title"],
```

Comment:
if you call `symbolize_keys!` on the hash, then all the keys will be symbols, and you won't have to do the `symbol || string` logic over and over again